lib/src/theme/wolt_modal_sheet_default_theme_data.dart

```diff
@@ -69,6 +69,9 @@ class WoltModalSheetDefaultThemeData extends WoltModalSheetThemeData {
   @override
   bool get enableDrag => true;
 
+  @override
+  bool get resizeToAvoidBottomInset => true;
```

Comment:
Defaulting to true here but the example explicitly set this property to `true` again.

Reply:
Fixed here: 20e03259844c0fadea3e3f9690add0246025aeb8